common/include/pcl/PCLPointCloud2.h

```diff
@@ -49,6 +51,51 @@ namespace pcl
   public:
     using Ptr = boost::shared_ptr< ::pcl::PCLPointCloud2>;
     using ConstPtr = boost::shared_ptr<const ::pcl::PCLPointCloud2>;
+
+    //////////////////////////////////////////////////////////////////////////
+    /** \brief Concatenate two pcl::PCLPointCloud2.
```

Comment:
Perhaps mention that this is an in-place variant right in the brief description. Useful to distinguish between overloads when auto-completing in IDEs.